packages/runtime/datastore/src/test/channelStorageService.spec.ts

```diff
@@ -17,9 +18,9 @@ describe("ChannelStorageService", () => {
             id: null,
             trees: {},
         };
-        const storage: Pick<IDocumentStorageService, "read"> = {
-            read: async (id: string) => {
-                assert.fail();
+        const storage: Pick<IDocumentStorageService, "readBlob"> = {
+            readBlob: async (id: string) => {
```

Comment:
Not sure if test code changed and now it's different, but prior implementation was just throwing here, now we have some implementation.